aicsimageprocessing/textureAtlas.py

```diff
@@ -207,9 +207,9 @@ def _is_valid_atlas(self, atlas):
         if atlas.atlas.shape is None:
             return False
         shape = atlas.atlas.shape
-        if self.dims.atlas_width != shape[1]:
+        if self.dims.atlas_width != shape[2]:
             return False
-        if self.dims.atlas_height != shape[2]:
+        if self.dims.atlas_height != shape[1]:
```

Comment:
height and width getting swapped here?

Reply:
Yea. I tried to link to the actual line in the diff but yea. It definitely got swapped in the last version somehow